skbio/stats/power.py

```diff
@@ -850,7 +919,7 @@ def _compare_distributions(test, samples, counts=5, mode="ind", num_iter=1000):
         this may be useful when working with regression data where
         :math:`x_{1}, x_{2}, ..., x_{n}` maps to :math:`y_{1}, y_{2}, ... ,
         y_{n}`.
-    num_iter : int, optional
+    num_iter : unsigned int, optional
```

Comment:
I think we shouldn't use `unsigned int` in our docs talking about scalars. We should instead say `positive` or `non negative int` depending on what we want. `unsigned int` is incorrect because it doesn't exist as a type in python, and seems hard for users to understand what it means unless they've programmed in C/...

If the input is an array, and the `dtype` is `uint` it's fine to specify that.